### DISCUSSION TITLE
Is there a way to update Money fields by a percentage?

### DISCUSSION THREAD

**Discussion by DomSaunders73** [Innofactor/XrmToolBox.BulkDataUpdater#177] [Q&A] [answered]
I am trying to implement a price increase across all Price Lists.  The "Update Price" is new in D365 and works against Price List Item, but I then use Discount Lists for user banding.

I can update the Price List Item by +5%, but then also need to update the Discount List by +5% to make it work correctly.

I've got the correct records in bulk updater but wondered whether the Calculate Function could be used to +5% to the amount and amount_base columns?

Something like: ({amount} * 1.05) - but I just get Error: Not valid Money

Many thanks.

**Comment by rappen** [comment, marked as answer]
A tiny more than that you wrote...

Try this:
`{amount|<value><Math|*|1,05>}`

First the `<value>` to get only the value, not with "kr", "USD" or whatever.
Then you have to say that you want to use `Math`.
Then add *
Finally add what you multiply with.

See docs with a bit more info: https://jonasr.app/xrm-tokens/#formatting

**Comment by DomSaunders73** [comment]
Awesome!  Many thanks for that.

Is there any way to round the result to 2 decimal places?

So, 7.33 * 1.05 results in 7.696500 but I want it to round to 7.700000 and 8.431500 should round to 8.430000.

Many thanks.

**Reply by rappen** [reply]
Make it 2 decimals by multiplying with 105, then round, then divide with 100.

`{amount|<value><Math|*|105><Math|Round><Math|/|100>}`

**Reply by rappen** [reply]
Oh crap... I just found a bug in the Round feature... You probably need to wait for a new version.
I'm working on it! Stay tuned...

**Reply by DomSaunders73** [reply]
Thank you! I was just trying it and got the error!

**Reply by rappen** [reply]
Now it should be possible - download both Bulk Data Updater and XRM Tokens Runner, versions 1.2022.11.*
Please create new issues if you have more problems or ideas!

**Comment by DomSaunders73** [comment]
UPDATE: @rappen - the 1,05 actually multiplies it by 105.  Use 1.05 to increase by 5%.

**Reply by rappen** [reply]
Depends on which locale you have... I have a Swedish locale, you probably have something else :)
